ci/scripts/commit-changes-to-public-repo.sh

```diff
@@ -37,7 +37,7 @@ popd
 pushd "$PUBLIC_REPO_LOCAL_NAME"/operators/"$OPERATOR_NAME"
 
 echo "Creating a new feature branch from the upstream main"
-git remote add upstream https://github.com/"$OWNER"/"$REPO".git
+git remote add upstream https://github.com/"$OWNER"/"$REPO".git 2>/dev/null || echo "Remote upstream already exists"
```

Comment:
Not entirely sure, but as you have set -e at the top, would adding an existing remote not terminate the entire script, as it is not existing with 0?